File: dev-docs/snippets/events/document-closed.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Fires when a document is closed (unsubscribed) by this client.
export function documentClosed(client: JsBaoClient) {
  // #region example
  client.on("documentClosed", (payload) => {
    console.log("closed", payload.documentId);
  });
  // #endregion example
}

File: dev-docs/snippets/events/document-closed.swift
import JsBaoClient

// Fires when a document is closed (unsubscribed) by this client.
func documentClosed(client: JsBaoClient) {
  // #region example
  let sub = client.events.on(.documentClosed) { (event: DocumentClosedEvent) in
    print("closed", event.documentId)
  }
  // #endregion example
  _ = sub
}
